packages/react-menu/package.json

```diff
@@ -21,9 +21,9 @@
     "just": "just-scripts",
     "lint": "just-scripts lint",
     "start": "just-scripts dev:storybook",
-    "start-test": "just-scripts jest-watch",
-    "test": "just-scripts test",
-    "update-snapshots": "just-scripts jest -u"
+    "start-test": "echo \"This is DEPRECATED instead use 'test --watch'\" && just-scripts jest-watch",
+    "test": "jest",
+    "update-snapshots": "echo \"This is DEPRECATED instead use 'test -u'\" && just-scripts jest -u"
```

Comment:
nit: I see `-u` and `--watch` let's be consistent with options ?

Reply:
-u -> --updateSnapshots makes sense when using it in package.json scripts 

But as i said it's a nit